### PR TITLE
Add Permissions For Bulkrax

### PR DESCRIPTION
**Pull request by orangewolf**
# Issue #
Ref https://github.com/scientist-softserv/britishlibrary/issues/227

# Scope #
- [x] put limits around the sidebar links
- [x] put limits in the controllers
- [ ] scope exporter for allowed works

# For Release Notes #

There are 2 new Ability methods that need to be added to `ability.rb`. If you are using the generator, these methods will be added for you. However, if you are upgrading Bulkrax, you will need to add the following to your `app/models/ability.rb` file above the `can_create_any_works?` method definition. Feel free to adjust the definitions as needed.

```
    def can_import_works?
      can_create_any_work?
    end
    
    def can_export_works?
      can_create_any_work?
    end
```

### CODE REVIEW

**Review by bkiahstroud — APPROVED** [review]

File: lib/generators/bulkrax/install_generator.rb [hunk 1]
```diff
@@ -55,6 +55,26 @@ def add_javascripts
     end
   end
 
+  def add_ability
+    file = 'app/models/ability.rb'
```
Comment (alishaevn) [inline lib/generators/bulkrax/install_generator.rb:59]: do we need to account for if this file doesn't exist at all?

**Review by alishaevn — CHANGES_REQUESTED** [review]
blocking the merge per my previous comment.

File: lib/generators/bulkrax/install_generator.rb (hunk 1 above)
Comment (bkiahstroud) [reply lib/generators/bulkrax/install_generator.rb:59]: [Hyrax expects the file to exist](https://github.com/samvera/hyrax/blob/bf5df6f9d47ff04dc5d103b5922bcb80a9bb821c/lib/generators/hyrax/install_generator.rb#L104-L110), so I think we would be okay to as well

Comment (bkiahstroud) [reply lib/generators/bulkrax/install_generator.rb:59]: Do you know of / have a case where the file doesn't exist?

Comment (alishaevn) [reply lib/generators/bulkrax/install_generator.rb:59]: ah ok. no, I don't know of any instance where it wouldn't...just wanted to make sure our bases were covered. 🙂

**Review by alishaevn — APPROVED** [review]
approving per kiah's responses to my question.

File: lib/generators/bulkrax/install_generator.rb (hunk 1 above)
Comment (bkiahstroud) [reply lib/generators/bulkrax/install_generator.rb:59]: It's a completely valid question, [we just ran into that issue in the Actions repo](https://github.com/scientist-softserv/actions/commit/9aac178a68bb9d0ee5e3eccb11899b5da7025f80) last week. So, better safe than sorry 🙂

**Pull request merged**